reporting/sendMail.js

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  logger: true,
+  auth: {
+    user: 'cypress.tests.results@gmail.com',
+    pass: 'tich vvpa eahv ohoi',
```

Comment:
sekurity huehue. Mozna to dodac do zmiennych na githubie, ale tylko ty masz do tego uprawnienia zdaje sie